src/interaction/inspector.jl

```diff
@@ -310,6 +310,10 @@ function on_hover(inspector)
                     should_clear = false
                     break
                 end
+            elseif (plotfunc(plt) === text && plt.parent.parent == inspector.plot) || (plotfunc(plt) === mesh && plt.parent == inspector.plot)
+                # prevent tooltip flickering when mouse hovers over the current inspectors tooltip
+                should_clear = false
+                break
```

Comment:
Would like some feedback from the devs about this case. It seems to work in practice but blindly calling `plt.parent.parent` seems a bit fragile. Is there a better way to determine if the mouse is hovering over a tooltip bubble?

Reply:
Yea it would be better to check this we other way around. i.e. compare with the child plots in `inspector.plot.plots`. That way you have a consistent object at least.

Also can you describe a bit more what kind of flickering this is supposed to prevent? Maybe with some example code/video?

Reply:
Here's flickering behavior I'm referring to:

https://github.com/user-attachments/assets/9184a2f9-e408-4d97-abfa-de16ab96a6a8

Reply:
Thanks for the suggestion, I flipped it to compare against child plots.

Reply:
It looks like `pick_sorted` doesn't return anything that's occluded, which is desirable in general. However, in this case it would be helpful if `pick_sorted` was able to return plots that are occluded by the tooltip. Then it would be easy to prevent flickering and still update the tooltip based on what's occluded by it.

Reply:
Yea I think I went through these considerations/problems before and settled on giving picking some range and hoping for the best.

Picking occluded things is in some sense impossible with the current implementation. Picking works by "drawing" a plot id + vertex index to a buffer while rendering and reading that when a picking function is called. That means you only have one value per pixel, one thing you can pick.

To address the flickering issue on a more fundamental level I see two options - write a ray based picking system or allow plots to opt out of picking during rendering.

For ray based picking you'd essentially want
```julia
for scene in all_scenes
    if !is_mouseinside(scene)
        continue
    ray = view_ray(scene) # depends on camera matrices
    for plot in scene.plots
        if collides_with_bbox(plot, ray) # because full intersection/collision checks can be very expensive
            idx, pos = intersection(plot, ray) # ~ position_on_plot()
            if # output marks valid intersection
                # return/collect collision data
```
A good chunk of that exists in ray_casting.jl already. However bounding boxes would probably become a bit of a pain point. You'd want to lazily cache them so that they one get recalculated when they could have changed and are needed.

The other option should be much less work on a surface level - add a generic attribute and effectively add `if (pickable)` to a few shaders in GLMakie and WGLMakie. On a detail level it gets more complicated though, because drawing order matters. If the occluded plot is drawn first this should work, otherwise not. Messing with the drawing order in GLMakie is fairly likely to produce some artifacts. Though it might also be fine if we just set `overdraw = true` for tooltips. 

Either way, these things shouldn't happen here. Maybe instead of locking the tooltip when its hovered it's better to increase the default picking `range` so that things outside the tooltip can be detected. That is more or less the original bandaid fix for this anyway.

Reply:
Those are all good points. I'll remove the tooltip flicking change from this PR and defer the proper solution to a different time.